frontend/webpack.dev.js

```diff
@@ -0,0 +1,36 @@
+const path = require('path');
```

Comment:
I'm not sure I totally understand the need of these two split webpack configs, I see some differences here and there but in the end both seem to target development. I see one enables `HotModuleReplacementPlugin`, is that so we have two ways of running things? if so I'm all good with that but I'd document these files a bit.

Reply:
My bad, there's some accidental file adding going on here ...